### PR TITLE
Add provisioner template for local storage

### PR DESCRIPTION
**Pull request by jsafrane**
Local volumes are alpha feature of 3.7 and  need some example how to set it up.

Based on https://github.com/kubernetes-incubator/external-storage/tree/master/local-volume, tailored for OpenShift.

I renamed original local-storage-examples to hostpath-examples.

### CODE REVIEW

**Comment by mfojtik** [comment]
Please use YAML for all templates/examples.

**Comment by jsafrane** [comment]
@mfojtik, it is yaml. Only existing files (moved across directories) are json.

**Comment by jsafrane** [comment]
@rootfs, please review

/assign @soltysh 
PTAL and approve

File: examples/storage-examples/local-examples/README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+## Alpha disclaimer
+
+Local Volumes are alpha feature in 3.7. It requires several manual steps to
+enable, configure and deplouy the feature. It may be reworked in the furute and
+it will be probably automated by openshift-ansible.
+
+
+## Overview
+Local volumes are PersistentVolumes representing local mounted filesystems.
+In future they may be extended to raw block devices.
```
Comment (rootfs) [inline examples/storage-examples/local-examples/README.md]: in the future

**Comment by rootfs** [comment]
minor nit, looks good to me

File: examples/storage-examples/local-examples/README.md (hunk 1 above)
Comment (jsafrane) [reply examples/storage-examples/local-examples/README.md]: fixed, PTAL

**Comment by jsafrane** [comment]
/retest
#16654

**Comment by jsafrane** [comment]
/retest
#17263

**Comment by jsafrane** [comment]
/retest
#17263 again

**Comment by jsafrane** [comment]
/retest
did I break anything?

**Comment by stevekuznetsov** [comment]
@jsafrane to see what the next steps are for your PR, view the Submit Queue status:

```
Submit Queue — PR does not have approved label. 
```

**Comment by jsafrane** [comment]
/retest
#15630

**Pull request merged**